component-maps.yml

```diff
@@ -173,6 +173,13 @@ git:
     - mender-deviceconnect
     release_component: true
 
+  devicemonitor:
```

Comment:
Missing similar entries for `docker_image` and `docker_container` sections

Reply:
right! fixed! sorry, forgot to scroll down this file.